CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## v0.6.0
 
 - [filesystem] added the menu item `Upload Files...` to easily upload files into a workspace
+- [preferences] changed signature for methods `getProvider`, `setProvider` and `createFolderPreferenceProvider` of `FoldersPreferenceProvider`.
```

Comment:
breaking changes should go in its own section, see below for 0.5.0

It would be nice also cover what for they were changed, see examples in breaknig changes for other releases